### PR TITLE
Web 643/Image download + Tooltip

### PR DESCRIPTION
**Pull request by ShreyaThapa**
- Used react-tippy for the Tooltip in images. https://bundlephobia.com/result?p=react-tippy@1.3.4
- Added a Tooltip component
- Added a ImageDownload component that wraps images that need a download link

### CODE REVIEW

File: components/base/Tooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+          }
+          .tippy-popper[x-placement^="right"] [x-arrow] {
+            border-right-color: ${GREY_8};
+          }
+          .tippy-popper[x-placement^="left"] [x-arrow] {
+            border-left-color: ${GREY_8};
+          }
+        `}
+      ></Global>
+      <ReactTooltip
```
Comment (ShreyaThapa) [inline components/base/Tooltip.tsx:53]: I've only included a few props from what's available in the library. We'll likely only need these few for our purposes, but we can add support for more if need be.

**Review by sausman — APPROVED** [review]

File: components/base/ImageDownload.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        <DownloadIconWrapper>
+          <Tooltip text={tooltipText} hasArrow position="bottom" distance={11}>
+            <a href={children} download={downloadFileName || ""}>
+              <DownloadIcon />
+            </a>
+          </Tooltip>
+        </DownloadIconWrapper>
+      </DownloadDiv>
+      <ImageDiv>
+        <img src={children} alt="" />
```
Comment (Finholt) [inline components/base/ImageDownload.tsx:52]: Can we give this a `max-width: 100%` or something so it shrinks on smaller mobile screens?

Comment (ShreyaThapa) [reply components/base/ImageDownload.tsx:52]: Good point! Totally overlooked that 😬

Comment (ShreyaThapa) [reply components/base/ImageDownload.tsx:52]: This [commit](https://github.com/Dwolla/dev-portal/pull/48/commits/6a8d1f2db59c9945e35f9aed498cadcb9599b596) should take care of that!

**Review by Finholt — APPROVED** [review]

**Pull request merged**